client/src/App/Components/Home.js

```diff
@@ -1,25 +1,25 @@
-import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
-import Button from './Button/Button';
+import React from "react";
+import { Link } from "react-router-dom";
+import Button from "./Button/Button";
 
-
-
-class Home extends Component {
-  render() {
-    return (
+const Home = () => {
+  return (
     <div className="App">
       <h1>Project Home</h1>
-      {/* Link to List.js */}
-      <Link to={'./list'}>
-        <button variant="raised">
-            My List
-        </button>
+      <Link to="./list">
+        <Button className="green"> ABOUT THE APP </Button>
```

Comment:
You shouldn't write uppercase text directly inside HTML/JSX elements or components, because screen readers will read it out letter by letter. Better practice is to use the ```text-transform: uppercase``` CSS property.